marray/tests/test_marray.py

```diff
@@ -417,11 +417,41 @@ def test_statistical_array(f_name, keepdims, xp=np, dtype='float64', seed=None):
 
 # Use Array API tests to test the following:
 # Creation Functions (same behavior but with all-False mask)
-# Data Type Functions (only `astype` remains to be tested)
-# Elementwise function `clip` (all others are tested above)
 # Indexing (same behavior as indexing data and mask separately)
 # Manipulation functions (apply to data and mask separately)
 
+
+@pytest.mark.filterwarnings('ignore::numpy.exceptions.ComplexWarning')
+@pytest.mark.parametrize('dtype_in', dtypes_all)
+@pytest.mark.parametrize('dtype_out', dtypes_all)
+@pytest.mark.parametrize('copy', [False, True])
+def test_astype(dtype_in, dtype_out, copy, xp=np, seed=None):
+    mxp = marray.masked_array(xp)
+    marrays, masked_arrays, seed = get_arrays(1, dtype=dtype_in, seed=seed)
+
+    if dtype_in != dtype_out and not copy:
+        pytest.mark.skip("Can't change type without copy.")
```

Comment:
does this actually do anything? I believe we need to use `pytest.skip` instead:
```suggestion
        pytest.skip(reason="Can't change type without copy.")
```

Reply:
No, it was a typo, and I guess it wasn't needed.